### PR TITLE
[ipa] Capture more cert information and other fixes

### PR DESCRIPTION
**Pull request by sbradley7777**
More cert information is captured, the "ipa-replica-manage"
commands removed because they would timeout because
a password was needed, sanitize some collected data,
and some logs with wildcards were not properly collected
is now fixed.

Fixes: #754

Signed-off-by: Shane Bradley <sbradley@redhat.com>

### CODE REVIEW

File: sos/plugins/ipa.py [hunk 1]
```diff
@@ -56,34 +57,33 @@ def setup(self):
...
-
         self.add_forbidden_path("/etc/dirsrv/slapd-*/key*")
         self.add_forbidden_path("/etc/dirsrv/slapd-*/pin.txt")
         self.add_forbidden_path("/etc/dirsrv/slapd-*/pwdfile.txt")
-
         self.add_forbidden_path("/etc/named.keytab")
 
         self.add_cmd_output([
             "ls -la /etc/dirsrv/slapd-*/schema/",
-            "ipa-getcert list",
```
Comment (battlemidget) [inline sos/plugins/ipa.py]: Why remove this line? If it worked on older distributions and those are still supported we shouldn't remove this.

Comment (bmr-cymru) [inline sos/plugins/ipa.py]: If the user doesn't have a valid kerberos ticket for the replication user It tries to do interactive auth (user/pass) at the terminal and hits the command timeout.

Comment (battlemidget) [inline sos/plugins/ipa.py]: ah ok thanks for the clarification

**Comment by bmr-cymru** [comment]
Merged as commit a30a418

**Pull request closed without merging**